pkg/util/test/util.go

```diff
@@ -3537,7 +3537,7 @@ func ValidateTelemetryV2Disabled(cluster *corev1.StorageCluster, timeout, interv
 func ValidateTelemetryV1Enabled(pxImageList map[string]string, cluster *corev1.StorageCluster, timeout, interval time.Duration) error {
 	logrus.Info("Validate Telemetry components are enabled")
 	opVersion, _ := GetPxOperatorVersion()
-	validateMetricsCollector := opVersion.GreaterThanOrEqual(opVer1_11)
+	validateMetricsCollector := opVersion.LessThan(opVer1_10)
```

Comment:
for v1 there's no difference, so can be consistent with v2.

Reply:
We do deploy metrics collector for operators 1.9 or below though